digiroad2-oracle/src/test/scala/fi/liikennevirasto/digiroad2/dao/linearasset/OracleSpeedLimitDaoSpec.scala

```diff
@@ -96,7 +97,7 @@ class OracleSpeedLimitDaoSpec extends FunSuite with Matchers {
 
       val speedLimits = dao.getSpeedLimitLinksByRoadLinks(roadLinks)
 
-      speedLimits._1.map(_.id) should equal(Seq(300103))
+      speedLimits._1.size should equal(1)
```

Comment:
Isn't enough validate only the size, if the size is 1 you should validate the id using head